### ISSUE TITLE
Admin project styles don't work with other from Development environment variable

### ISSUE THREAD

**Issue by Diponchik** [skoruba/Duende.IdentityServer.Admin#65]
Hello there. I was trying to run an admin project with my "Local" ASPNETCORE_ENVIRONMENT, but I discovered, that styles don't work on this environment and after I changed it back to development, then it started working.

Steps to reproduce: Change ASPNETCORE_ENVIRONMENT to something else ("Local" for example) and run the project. Styles should be missed.

![image](https://user-images.githubusercontent.com/25414988/158771790-398365d2-f7c0-4d76-ad30-5aecd30e3573.png)

**Comment by JacksonHamner** [comment]
We are seeing this in our project too, any idea what might be causing this?

**Comment by selangley-wa** [comment]
It appears to be something strange with how the staticwebassets for the Skoruba.Duende.IdentityServer.Admin.UI NuGet package are handled / processed when running in a non-development environment.  

For my custom Duende.IdentityServer.Admin project generated using the template, if I:

1. Move the directory:

{$USERPROFILE}\.nuget\packages\skoruba.duende.identityserver.admin.ui\1.1.0\staticwebassets

to {$MyTemplateGeneratedProjectName}\src\{$MyTemplateGeneratedProjectName}.Admin\wwwroot

2. Remove the importing of the staticwebassests as a dependency for the  Skoruba.Duende.IdentityServer.Admin.UI NuGet package, which I achieved in a "hacky" way by deleting the line:

  <Import Project="Microsoft.AspNetCore.StaticWebAssets.props" />
  
  from the file:
  
  {$USERPROFILE}\.nuget\packages\skoruba.duende.identityserver.admin.ui\1.1.0\build\Skoruba.Duende.IdentityServer.Admin.UI.props
  
3. Finally, rebuild  my custom Duende.IdentityServer.Admin project.

Then the styles for the Admin project are found successfully when running in an non-development configuration.

Perhaps the manner in which the staticwebassets are being packaged can be improved upon?

The only real documentation I found from Microsoft on this is the following blog post:

https://devblogs.microsoft.com/dotnet/build-client-web-assets-for-your-razor-class-library/

and this documentation page:

https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-6.0&tabs=visual-studio
